name: CD2
on:
  workflow_run:
    workflows: [CI]
    types: 
      - completed
      
jobs:
  execute:
#     if:  ${{ github.event.workflow_run.conclusion == 'failure' }}
    runs-on: ubuntu-latest
    steps: 
      - name: get_job_steps
        uses: actions/github-script@v6
        with:
          script: |
            let workflowRunJobs = await github.rest.actions.listJobsForWorkflowRunAttempt({
              owner: context.repo.owner,
              repo: context.repo.repo,
              run_id: context.payload.workflow_run.id
              attempt_number: context.payload.workflow_run.run_attempt
            })
            console.log(workflowRunJobs);
            let url = context.payload.workflow_run.jobs_url;
            let jobsOutput = url.split(".com")[1];
            console.log(jobsOutput);
            let stepsOutput = jobsOutput["steps"];
            console.log(stepsOutput);
#            core.setOutput('stepsOutput', stepsOutput);
            
            
#       - name: find_failed_step
#         runs: |
#           ${{ steps.get_job_steps.outputs.stepsOutput
        
        
#       - name: resolve_error
